src/main/java/VisualLogic/ElementPalette.java

```diff
@@ -160,16 +152,20 @@ public String getEndung(String fileName) {
 
     public List<String> getSortDefinitionFile(File file) {
         List<String> list = new ArrayList<>();
-        try {
-            BufferedReader input = new BufferedReader(new FileReader(file.getAbsolutePath() + "/" + "sort.def"));
+
+        try (BufferedReader input = new BufferedReader(new FileReader(file.getAbsolutePath() + "/" + "sort.def"))) {
             String inputString;
             while ((inputString = input.readLine()) != null) {
                 list.add(inputString);
             }
-            input.close();
-        } catch (Exception ex) {
-            // System.out.println("FEHLER : "+ex.getMessage());
+
+        } catch (FileNotFoundException e) {
+            org.tinylog.Logger.error(e);
+        } catch (IOException ioException) {
```

Comment:
Why not combine both exceptions into one path?

Reply:
addresed here 0aaf39a8